readthedocs/donate/templates/donate/promo_detail.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block title %}{% trans "Promo Detail" %}{% endblock %}
+
+{% block content %}
+
+<h1> Promo Results </h1>
+
+{% for promo in promos %}
+
+<h3>
+Results for {{ promo.name }} ({{ promo.analytics_id }}) over last {{ days }} days.
+</h3>
+
+<div class="example" style="width: 30%;">
+<a href="{{ promo.link }}"><img width=120 height=90 src="{{ promo.image }}"></a>
+<br>
+{{ promo.text }}
+
+</div>
+
+<br>
+
+<table>
+  <tr>
+    <th><strong>Day (UTC)</strong></th>
+    <th><strong>Views</strong></th> 
+    <th><strong>Clicks</strong></th>
+    <th><strong>CTR</strong></th>
+  </tr>
+  {% for day in promo.impressions.all|slice:days_slice %}
+  <tr>
+    <td>{{ day.date }}</td>
+    <td><code>{{ day.views }}</code></td> 
+    <td><code>{{ day.clicks }}</code></td>
+    <td><code>{{ day.click_ratio }}%</code></td>
+  </tr>
+  {% endfor %}
+  <tr>
+    <td><strong>Total (over all time)</strong> </td>
+    <td><strong>{{ promo.total_views }}</strong></td> 
+    <td><strong>{{ promo.total_clicks }}</strong></td>
+    <td><strong>{{ promo.total_click_ratio }}%</strong></td>
```

Comment:
Same for styling here. This should perhaps be a rule for `table > tr.table-totals` or somesuch.